README.md

```diff
@@ -158,10 +158,10 @@ The following image types are currently available via the `--type` argument:
 
 | Image type            | Target environment                                                                    |
 |-----------------------|---------------------------------------------------------------------------------------|
-| `ami`                 | [Amazon Machine Image](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) |
+| `ami`, `raw`          | [Amazon Machine Image](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) |
 | `qcow2` **(default)** | [QEMU](https://www.qemu.org/)                                                         |
 | `vmdk`                | [VMDK](https://en.wikipedia.org/wiki/VMDK) usable in vSphere, among others            |
-| `anaconda-iso`        | An unattended Anaconda installer that installs to the first disk found.               |
+| `iso`, `anaconda-iso` | An unattended Anaconda installer that installs to the first disk found.               |
```

Comment:
Conceptually `iso` is a now-deprecated alias for `anaconda-iso`; I think we should continue to omit it from the help output, or even just remove it entirely now.